FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c

```diff
@@ -1158,7 +1158,7 @@ void test_coverage_prvCreateIdleTasks_name_too_long( void )
     xIdleTask = ( TCB_t * ) xIdleTaskHandles[ 0 ];
 
     /* Test Verifications */
-    TEST_ASSERT_EQUAL_STRING_LEN( configIDLE_TASK_NAME,
+    TEST_ASSERT_EQUAL_STRING_LEN( "IDLE long 0",
```

Comment:
```suggestion
    TEST_ASSERT_EQUAL_STRING_LEN( "IDLE long n",
```

Reply:
The tests will be fixed after this PR is merged - https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1203.